spec/feature/items_view_spec.rb

```diff
@@ -2,7 +2,18 @@
 
 describe "the item view", type: :feature do
   let(:item) do 
-    Item.create(title: "Possum Stew", description: "Savory stew made with fresh possum", image: "www.possum.com", item_status_id: 1, price: 2.35)
+    Item.create(title: "Possum Stew",
+               description: "Savory stew made with fresh possum", 
+               image: "www.possum.com", 
+               item_status_id: 1, 
+               price: 2.35)
+  end
+  let(:item2) do 
+    Item.create(title: "Possum2 Stew",
+               description: "Savory stew made with fresh possum", 
+               image: "www.possum.com", 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.